### PR TITLE
Update and improve UI

### PR DESCRIPTION
**Pull request by Sense101**

### CODE REVIEW

File: client/src/app/app.module.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+button {
```
Comment (martinkadlec0) [inline client/src/app/app.module.css:1]: global styles should be in client/src/index.css not here.

File: client/src/app/app.module.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+input {
+	border: 0.3vh solid #000;
+}
+input:focus-visible {
+	outline-offset: 0;
+	outline: 0.5vh solid #333;
+	border-radius: 0.5vh;
+}
+
+::-webkit-scrollbar {
```
Comment (martinkadlec0) [inline client/src/app/app.module.css:21]: These will not fully work in FF. It is okay to have not have it FF but we should check we it loooks okay-ish.

**Review by martinkadlec0 — CHANGES_REQUESTED** [review]

File: client/src/app/app.module.css (hunk 1 above)
Comment (Sense101) [reply client/src/app/app.module.css]: Done.

File: client/src/app/app.module.css (hunk 2 above)
Comment (Sense101) [reply client/src/app/app.module.css]: I've tested it on firefox, can confirm that the scrollbar stays as the default on firefox. However, it doesn't cause any issues, and the default scrollbar actually looks quite nice.

**Pull request merged**